docs/reference/snapshot-restore/restore-snapshot.asciidoc

```diff
@@ -1,187 +1,523 @@
 [[snapshots-restore-snapshot]]
 == Restore a snapshot
 
-////
+This guide shows you how to restore a snapshot. Snapshots are a convenient way
+to transfer data between clusters. You can also restore a snapshot to recover
+indices and data streams after deletion or a hardware failure.
+
+In this guide, you'll learn how to:
+
+* Retrieve available snapshots
+* Restore an index or data stream from a snapshot
+* Restore an entire cluster
+* Monitor the restore operation
+* Cancel an ongoing restore
+
+This guide also provides tips for <<restore-different-cluster,restoring to
+another cluster>> and <<troubleshoot-restore,troubleshooting common restore
+errors>>.
+
+[discrete]
+[[restore-snapshot-prereqs]]
+=== Prerequisites
+
+include::apis/restore-snapshot-api.asciidoc[tag=restore-prereqs]
+
+[discrete]
+[[restore-snapshot-considerations]]
+=== Considerations
+
+When restoring data from a snapshot, keep the following in mind:
+
+* If you restore a data stream, you also restore its backing indices.
+
+* You can only restore an existing index if it's <<indices-close,closed>> and
+the index in the snapshot has the same number of primary shards. 
+
+* You can't restore an existing open index. This includes
+backing indices for a data stream.
+
+*  The restore operation automatically opens restored indices, including backing
+indices.
+
+* You can restore only a specific backing index from a data stream. However, the
+restore operation doesn't add the restored backing index to any existing data
+stream.
+
+[discrete]
+[[retrieve-snapshot]]
+=== Retrieve a snapshot
```

Comment:
"Retrieve" isn't quite the right word IMO, it suggests you're retrieving all the data the snapshot contains but really you're just getting a bit of information about its contents.